client/trino-client/src/main/java/io/trino/client/JsonResponse.java

```diff
@@ -36,6 +38,7 @@
     private final int statusCode;
     private final String statusMessage;
     private final Headers headers;
+    @Nullable
```

Comment:
nit: We usually prefer `Optional<...>`. It causes an extra object allocation, but this is merely a single object per a single round trip of communication, shouldn't be a big deal.

Reply:
True, I don't think it will matter much either way. I was aiming to keep consistency with the surrounding code where `exception` is nullable and `value` is nullable (either because the response was a `null` literal, or because there was an exception). I'm happy to change this to `Optional<String>` directly if you would prefer.

Reply:
Hmm. I'm fine with keeping it consistent with the surrounding code.